packages/web-components/src/components/va-privacy-agreement/test/va-privacy-agreement.e2e.ts

```diff
@@ -122,9 +124,16 @@ describe('va-privacy-agreement', () => {
       '<va-privacy-agreement />',
     );
     const analyticsSpy = await page.spyOnEvent('component-library-analytics');
-    const vaCheckbox = await page.find('va-privacy-agreement >>> va-checkbox');
+    const input = (
```

Comment:
This test was never even really working as intended because the click was on `va-checkbox` instead of the `input` so I have updated it.